package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "waterdataui-assets",
+  "version": "1.0.0",
```

Comment:
The version should probably match what is in .bumpversion.cfg, currently 0.1.0dev0. Can we use bumpversion to update this version, too?

Reply:
Hadn't considered that - would be good to do. The dev0 suffix might not be valid with the semantic versioning npm expects - although, we won't actually be publishing to the npm repository. Will investigate.